code/obj/cryotron.dm

```diff
@@ -128,6 +128,9 @@
 				return
 			if (thePerson.loc == firstLoc)
 				step(thePerson, SOUTH)
+			for (var/obj/O in src.loc) // dropped stuff & whatever spawned under them
```

Comment:
This will only include the south west corner of the cryotron, is that where the objects spawn?

Reply:
cryotron does some special setup in New() to make its loc where it spawns players:
https://github.com/goonstation/goonstation/blob/master/code/obj/cryotron.dm#L29-L40

Reply:
oh neat!